qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferToPDFConverter.groovy

```diff
@@ -144,25 +145,25 @@ class OfferToPDFConverter implements OfferExporter{
         // Set the start offer position
         def itemPos = 1
         // Create the items in html in the overview table
-        offer.items.each {item ->
+        offer.items.each { item ->
             htmlContent.getElementById("product-items")
                     .append(ItemPrintout.itemInHTML(itemPos++, item))
         }
     }
 
     void setPrices() {
-        final totalPrice = offer.totalPrice
-        final taxes = offer.taxes
-        final netPrice = offer.netPrice
+        final totalPrice = Currency.getFormatterWithSymbol().format(offer.totalPrice)
+        final taxes = Currency.getFormatterWithoutSymbol().format(offer.taxes)
+        final netPrice = Currency.getFormatterWithoutSymbol().format(offer.netPrice)
 
-        htmlContent.getElementById("total-costs").text(totalPrice.toString()+" €")
+        htmlContent.getElementById("total-costs").text(totalPrice)
```

Comment:
why is the symbol removed here?

Reply:
The symbol is now part of the function getFormatterWithSymbol() :) I thought its cleaner to keep it there and not to hardcode it here